### DISCUSSION TITLE
Auto close issues on merge

### DISCUSSION THREAD

**Discussion by FlorianEagox** [devicons/devicon#749] [Ideas]
Will issues with PRs referencing them get closed automatically?
There are lots of open issues, could some of them be kicking around from old updates that are yet to be closed?
It might be a good idea to use one of those actions and if a PR references an issue, make it in-progress, and if it gets merged, close it automatically. If it already closes them, cool, but marking them in-progress would make it easier for people to choose which icons to do next.

**Comment by Thomas-Boi** [comment]
I think this is a good idea. There are some actions that we can use to label issues so it won't be too hard to do this. I'll convert this to an issue since it's a feature request.

**Comment by pipahaha** [comment]
@[word counter](https://wordcounter.tools):
I think it makes sense, thank you!
